### ISSUE TITLE
List of scenes to activate does not use user provided names

### ISSUE THREAD

**Issue by austinmroczek** [Z-Wave-Me/zwave-smarthome#227]
In any application which provides the "List of scenes to activate" option, the drop down list of scene names does not match the names the user provided for each scene.

I create a number of scenes and provided a useful name for each one, such as "My First Scene".  I then open an application (for example Schedule) to use rules of some sort to activate the scene.  I click "Add new item" button under the "list of scenes to activate" option.  A drop down list of scenes is provided, however each of them is named "Light Scene [#]" 

![list of scenes](https://cloud.githubusercontent.com/assets/21700514/24785313/b036036e-1b0d-11e7-9db0-49a9d9f69295.png)

This is the same for every application that provides the drop down list of scenes to activate.

The drop down list should use the names the user created for each scene.

UI version	1.7.0-RC-9

**Comment by ronie-z-wave** [comment]
Hi austinmroczek,

did you change the name of the instance - list of active scenes - or the name of the by the instance created widget under elements view?
Usually the scene instance creates a widget for the scene that could be triggered by the apps. Ususally the names of the available scene widgets are listed in app configuration dropdowns. 

If that isn't the case then indeed it seems to be a bug.

**Comment by austinmroczek** [comment]
I created a scene using the "Scenes" app, and named it "My First Scene" inside that app.

I now see that the Lighting Scene 10 "element" is separate from the My First Scene "widget" as you call it, and says that it was created by the "My First Scene" scene app.

Is this the intended function?  It is not very intuitive, at least to me.  When I created the scene and defined the name for it, I expected that name to show up in the drop down list of scenes.

Perhaps instead of a bug, this is a feature request.  When the "Scene" app creates a new widget with name "Scene X" and creates the scene element, the widget should give the element the same name "Scene X".  The other option would be to directly pull the Scene widget name instead of using the Element name.
